python/vast/utils/arrow.py

```diff
@@ -0,0 +1,26 @@
+import ipaddress
+
+import pyarrow as pa
+
+
+class IPAddressType(pa.PyExtensionType):
+    def __init__(self):
+        pa.PyExtensionType.__init__(self, pa.binary(16))
```

Comment:
Not sure about the API, but don't you need to put the extension type identifier `vast.address` somewhere?

Reply:
Good point, adding that.